assets/styles/app.scss

```diff
@@ -5,16 +5,39 @@
     padding: 0;
 }
 
+/* ------------------------- body ------------------------- */
+
+body {
+    background-color: $secondary-color;
+}
+
 /* ------------------------- navbar ------------------------- */
 
-.nav-containt {
-    list-style: none;
+.nav-container {
+    list-style-type: none;
     display: flex;
     flex-direction: row;
     justify-content: space-evenly;
-    margin-top: 1rem;
     font-size: 30px;
     position: fixed;
     width: 100%;
     bottom: 0;
+    border-top: solid;
+    border-color: $golden-color;
+    border-width: 5px;
+    margin-top: 1rem;
+    height: 4rem;
```

Comment:
no fixed height: let the content drives the size

Reply:
Done !